### DISCUSSION TITLE
Daniel Stockman - Lerna

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#79] [Announcements]
## Intro

Andrew: Hello, welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin.

Justin: Hey everyone! Our guest today is Daniel Stockman.

## Questions

- [x] What experience did you have in open source before getting involved with lerna?
- [x] How did you learn of lerna, get involved, and eventually become the sole maintainer?
- [x] What was it like helming a project with such a big user base as basically the only man on the ship?
- [x] Were there things that still brought you joy from the project? 
- [x] What portion of the work lead you to the burnout you experienced with the project?
- [x] What advice would you give an engineer on how to spot burn out coming? Do you have any tips on how to get through it and out of it?
- [x] For awhile there it seemed like the project was going to die, but recently you announce that you're handing over the reigns to nrwl. Why were you initially hesitant to transfer maintenance to another person? What about nrwl changed that?
- [x] Do you plan to publish more packages or making more tools?

## Outro

Andrew: Well that’s it for this week’s episode of devtools.fm. Be sure to follow us on YouTube and wherever you consume your podcasts. Thanks for listening!

**Comment by hipstersmoothie** [comment]
## Research

https://github.com/lerna/lerna/issues/2703
https://github.com/lerna/lerna/issues/3121
https://github.com/evocateur

**Comment by hipstersmoothie** [comment]
## Tooltips

https://github.com/shuding/tilg
https://github.com/ericclemmons/click-to-component

**Comment by evocateur** [comment]
Great questions, very au courant. Looking forward to recording!

**Reply by evocateur** [reply]
## Tooltips

* https://github.com/sharkdp/bat
* https://github.com/sharkdp/fd

**Comment by just-be-dev** [comment]
tooltips:
- https://github.com/thepassle/astro-service-worker
- https://github.com/spacedriveapp/spacedrive
